CONTRIBUTING.md

```diff
@@ -34,25 +35,30 @@ git clone git@github.com:nextauthjs/next-auth.git
 cd next-auth
 ```
 
-2. Install packages and set up the developer application:
+2. Install packages:
 
 ```sh
-npm run dev:setup
+nvm use
```

Comment:
Instead of mentioning `nvm`, let's say "developing requires at least *version from .nvmrc file here."

Reply:
Fair point, but I think it needs to be the exact version of `node` in `.nvmrc` (not at least). For example, my version is v17 and I got this error if not switching to v16:
```
error root@0.0.0: The engine "node" is incompatible with this module. Expected version "^12.19.0 || ^14.15.0 || ^16.13.0". Got "17.3.0"
error Found incompatible module.
```